src/em/em.cpp

```diff
@@ -683,12 +694,19 @@ int em_t::send_frame(unsigned char *buff, unsigned int len, bool multicast)
         }
     }
 
+    //override destination and source mac addresses
+    sdu.setDestinationAlMacAddress({buff[0],buff[1],buff[2],buff[3],buff[4],buff[5]});
+    sdu.setSourceAlMacAddress({buff[6],buff[7],buff[8],buff[9],buff[10],buff[11]});
```

Comment:
This should do the same while avoiding having to scrutinize the indices or worry that one got changed
```suggestion
    sdu.setDestinationAlMacAddress({bytes, bytes + ETH_ALEN});
    sdu.setSourceAlMacAddress({buff+ETH_ALEN, bytes + (2*ETH_ALEN)});
```